pkg/pillar/types/newlogtypes.go

```diff
@@ -81,3 +84,22 @@ type NewlogMetrics struct {
 	DevMetrics logfileMetrics // Device metrics
 	AppMetrics logfileMetrics // App metrics
 }
+
+// GetTimestampFromGzipName - get timestamp from gzip file name
+func GetTimestampFromGzipName(fName string) (time.Time, error) {
```

Comment:
Then this should also replace this func, I guess?
https://github.com/lf-edge/eve/blob/63c8aae0a04bfb4dc72172bd0aff50d17e9142ec/pkg/pillar/cmd/loguploader/loguploader.go#L739

Reply:
exactly, I'm doing this in the main PR https://github.com/lf-edge/eve/pull/4413